timesketch/models/sketch.py

```diff
@@ -735,6 +742,11 @@ class InvestigativeQuestionConclusion(LabelMixin, StatusMixin, CommentMixin, Bas
     analysis = relationship(
         "Analysis", backref="investigativequestionconclusion", lazy="select"
     )
+    events = relationship(
+        "Event",
+        secondary=questionconclusion_event_association_table,
+        backref="conclusions", lazy="select"
+    )
```

Comment:
Do we need the backref when we have the `questionconclusion_event_association_table` table?

Reply:
I believe so:
- Without backref="conclusions", if you had an Event object, the only way to find the associated QuestionConclusion objects would be to query the QuestionConclusion table and filter based on the association table.
- With backref="conclusions", you can directly access the associated QuestionConclusion objects from an Event object.